test/Groups/gsets.jl

```diff
@@ -0,0 +1,67 @@
+@testset "G-sets of permutation groups" begin
+
+  # natural constructions (determined by the types of the seeds)
+  G = symmetric_group(6)
+  Omega = gset(G)
+  @test length(Omega) == 6
+  @test length(orbits(Omega)) == 1
+  @test istransitive(Omega)
+  @test ! isregular(Omega)
+  @test ! issemiregular(Omega)
+
+  Omega = gset(G, Set([Set([1, 2])]))  # action on unordered pairs
+  @test length(Omega) == 15
+  @test length(orbits(Omega)) == 1
+  @test istransitive(Omega)
+  @test ! isregular(Omega)
+  @test ! issemiregular(Omega)
+
+  Omega = gset(G, Set([[1, 2]]))  # action on ordered pairs
```

Comment:
Do we really need to insist on the seeds being a `Set`? Any collection resp. itererable should do it, too, shouldn't it? With `collect` we can still turn that into a `Vector` and that then into a Set